sqldelight-compiler/src/main/kotlin/app/cash/sqldelight/core/compiler/DatabaseGenerator.kt

```diff
@@ -118,14 +116,11 @@ internal class DatabaseGenerator(
   private fun forAdapters(
     block: (PropertySpec) -> Unit
   ) {
-    val queriesFile = sourceFolders
+    sourceFolders
       .flatMap { it.queryFiles() }
+      .flatMap { it.requiredAdapters }
```

Comment:
@AlecStrong testing something out here because I can't figure out why the integration tests are failing.

In the current code, only the first `SqlDelightQueriesFile`. Is that a bug, or is everything concatted into one file?